### ISSUE TITLE
Qualified columns

### ISSUE THREAD

**Issue by mbardelmeijer** [lorisleiva/laravel-search-string#33]
I saw that dfd92b8 introduced support for qualified columns which are a welcome addition. We we're already specifying qualified columns, and this breaks the existing behavior. The table name is added double:

```
protected $searchStringColumns = [
    'calls.identifier' => ['searchable' => true],
];
```

Error: `Unknown column 'calls.calls.identifier' in 'where clause'`

I think the table name only needs to be prefixed if not specified before?

**Comment by lorisleiva** [comment]
Hi there 👋 

Thanks for raising this. You're right, checking if the column already contains a `.` should fix this. I'm super busy on another project at the moment so feel free to PR something otherwise I'll take a look as soon as I have a moment. 🙂 

**EDIT**: Okay this was actually super simple to fix so it's fixed and released as `v1.1.1`. 🍺

**Comment by mbardelmeijer** [comment]
Awesome, thanks for the fix💪
